src/functions/keycloak/update-user-attributes/handler.ts

```diff
@@ -5,76 +5,78 @@ import { CreateSeveraApiService } from "src/services/severa-api-service";
+        statusCode: 400,
+        body: JSON.stringify({ message: "Missing or invalid parameters: 'id' or 'attribute'." })
+      };
+    }
 
-  if (!id || !attributes || typeof attributes !== "object") {
-    return {
-      statusCode: 400,
-      body: JSON.stringify({ message: "Missing or invalid parameters: 'id' or 'attribute'." }),
-    };
-  }
+    const keys = Object.keys(attributes);
+    if (!keys.every((key) => allowedKeys.includes(key))) {
+      return {
+        statusCode: 400,
+        body: JSON.stringify({ message: "Attributes contain invalid keys." })
+      };
+    }
 
-  const keys = Object.keys(attributes);
-  if (!keys.every(key => allowedKeys.includes(key))) {
-    return {
-      statusCode: 400,
-      body: JSON.stringify({ message: "Attributes contain invalid keys." }),
-    };
-  }
+    const api = CreateKeycloakApiService();
+    const severaApi = CreateSeveraApiService();
 
-  const api = CreateKeycloakApiService();
-  const severaApi = CreateSeveraApiService()
+    if (!Object.prototype.hasOwnProperty.call(attributes, attributeName)) {
+      return {
+        statusCode: 400,
+        body: JSON.stringify({ message: `Attribute ${attributeName} is missing in request body.` })
+      };
+    }
 
-  let severaUser: {email:string, guid:string } | null = null
+    const severaUser = await severaApi.getUserByEmail(email, attributes);
 
-  if (process.env.NODE_ENV === "development") {
-    severaUser = await severaApi.getTestUser();
-  } else {
-    severaUser = await severaApi.getUserByEmail(email, attributes);
-  }
-  if (!severaUser || !severaUser.email) {
-    return {
-      statusCode: 404,
-      body: JSON.stringify({ message: "Severa user not found." }),
-    };
-  }
+    if (!severaUser?.email || !severaUser?.guid) {
+      return {
+        statusCode: 404,
+        body: JSON.stringify({ message: "Severa user not found." })
+      };
+    }
 
-  if (!attributes.isActive) {
-    attributes.isActive = ["Active"];
-  }
-  attributes["severa-user-id"] = [severaUser.guid];
+    if (!attributes.isActive) {
+      attributes.isActive = ["Active"];
+    }
+    attributes["severa-user-id"] = [severaUser.guid];
 
-  await api.updateUserAttribute(id, attributes);
+    await api.updateUserAttribute(id, attributes);
 
-  await severaApi.getUserByEmail(severaUser.email, attributes);
-  
-  return {
-    statusCode: 200,
-    body: JSON.stringify({ message: "IsSeveraOptin has already been updated in Severa, and severaId will be added to Keycloak." }),
-  };
-} catch (error) {
-  return {
-    statusCode: 500,
-    headers: { "Content-Type": "application/json" },
-    body: JSON.stringify({ message: error.message, stack: error.stack }),
-  };
-}
+    await severaApi.getUserByEmail(severaUser.email, attributes);
+
+    return {
+      statusCode: 200,
+      body: JSON.stringify({
+        message: `Severa user updated with: ${JSON.stringify({ attributes })}, Keycloak attributes updated: ${JSON.stringify(attributes)}`
```

Comment:
As I said in a previous comment don't we get any kind of return values form the API's that we can return here, rather than just returning the attributes that the user provides. 
I think if there are bugs/ anything weird going on (now or in the future with further development) doing this will make them easier to spot